_posts/en/pages/2017-01-01-download.md

```diff
@@ -54,6 +54,91 @@ patient: "Check your bandwidth and space"
 # releasekeys: "Bitcoin Core Release Signing Keys"
 releasekeys: "Bitcoin Core Release Signing Keys"
 
+pgp_key_fingerprint: "PGP key fingerprint"
+verify_download: "Verify your download"
+verification_recommended: "Download verification is optional but highly recommended.  Click one of the lines below to view verification instructions for that platform."
+windows_instructions: "Windows verification instructions"
+macos_instructions: "MacOS verification instructions"
+linux_instructions: "Linux verification instructions (not for Ubuntu PPA)"
+ubuntu_ppa_instructions: "Ubuntu PPA verification instructions"
+download_release: "Click the link in the list above to download the release for your platform and wait for the file to finish downloading."
+download_checksums: "Download the list of cryptographic checksums:"
+cd_to_downloads: "Open a terminal (command line prompt) and Change Directory (cd) to the folder you use for downloads.  For example:"
+cd_example_linux: "cd Downloads/"
+cd_example_windows: >
+  cd %UserProfile%\Downloads
+
+verify_download_checksum: "Verify that the checksum of the release file is listed in the checksums file using the following command:"
+checksum_warning_and_ok: 'In the output produced by the above command, you can safely ignore any warnings and failures, but you must ensure the output lists "$(SHASUMS_OK)" after the name of the release file you downloaded.  For example:'
+obtain_release_key: "Obtain a copy of the release signing key by running the following command:"
+release_key_obtained: "The output of the command above should say that one key was imported, updated, has new signatures, or remained unchanged."
+verify_checksums_file: "Verify that the checksums file is PGP signed by the release signing key:"
+check_gpg_output: "Check the output from the above command for the following text:"
+line_starts_with: "A line that starts with:"
+complete_line_saying: "A complete line saying:"
+gpg_trust_warning: >
+  The output from the previous command may contain a warning that
```

Comment:
"The output from the previous command may contain..." was misleading to me. I interpreted it as the "command previous to the above command" for some reason? I suggest to just write "The output may contain..."

Reply:
Did s/previous/verify/ so that it now reads "The output from the verify command may contain..." so that it clearly refers to the `gpg --verify` command.   Thanks!